go/vt/mysqlctl/backup.go

```diff
@@ -396,10 +404,26 @@ func Restore(ctx context.Context, params RestoreParams) (*BackupManifest, error)
 		return nil, err
 	}
 
+	if handles := restorePath.IncrementalBackupHandles(); len(handles) > 0 {
+		params.Logger.Infof("Restore: applying %v incremental backups", len(handles))
+		for _, bh := range handles {
+			manifest, err := re.ExecuteRestore(ctx, params, bh)
+			if err != nil {
+				return nil, err
+			}
+			// Add backupTime and restorePosition to LocalMetadata
+			params.LocalMetadata["RestoredBackupTime"] = manifest.BackupTime
+			params.LocalMetadata["RestorePosition"] = mysql.EncodePosition(manifest.Position)
```

Comment:
Does this go into `_vt.local_metadata`? We are about to delete that table because it was being used primary by orchestrator integration (which has now been deleted in main).

Reply:
The tablet record in topo maybe the right place for this. The bigger question is: why do we want to record this information?

Reply:
@deepthi I'm happy to remove these lines -- I only added them for consistency with existing code, and am not using this data anywhere. If the existing code is to be removed, then there's no need for these two lines of code.

Reply:
Removed.